tests/smoke-tests/test-historic-compatibility.ts

```diff
@@ -0,0 +1,83 @@
+import "@moonbeam-network/api-augment";
+import { expect } from "chai";
+import { describeSmokeSuite } from "../util/setup-smoke-tests";
+import { NetworkTestArtifact, tracingTxns } from "../util/tracing-txns";
+import Bottleneck from "bottleneck";
+import { providers } from "ethers";
+
+const debug = require("debug")("smoke:historic-compatibility");
+const limiter = new Bottleneck({ maxConcurrent: 10, minTime: 100 });
+const httpEndpoint = process.env.HTTP_URL;
+
+describeSmokeSuite(`Verifying historic compatibility...`, async (context) => {
+  let traceStatic: NetworkTestArtifact;
+  before("Loading tracing static data", async function () {
+    const chainId = (await context.polkadotApi.query.ethereumChainId.chainId()).toString();
+    debug(`Running tracing tests against chainId ${chainId}.`);
+    traceStatic = tracingTxns.find((a) => a.chainId.toString() == chainId);
```

Comment:
I'm always afraid of `==` failing in the most glorious of ways. Would sleep better at night knowing it's a `===` :)

Reply:
hahaha, sure thing. Typescript allegedly fixes this, but i'm also ok to be sceptical of that claimed fact.